internal/apiserver/api/controllers/routers.go

```diff
@@ -0,0 +1,26 @@
+package api
```

Comment:
I suggest you put this file in another package as a sibling of controllers

Reply:
Could you explain in more detail?

Reply:
In my past experience, routers do not belong to controllers, they called controllers instead.
So I suggest put routers.go in a package named route which out of controllers

Reply:
It makes sense. However, routers are relevant to controllers that all belong to the API. I'd prefer to keep them in the same package `api`. What do you think?